cockatrice/src/dlg_register.cpp

```diff
@@ -62,9 +62,7 @@ DlgRegister::DlgRegister(QWidget *parent)
     countryEdit->setCurrentIndex(0);
     QStringList countries = settingsCache->getCountries();
     foreach(QString c, countries)
-    {
-        countryEdit->addItem(QPixmap(":/resources/countries/" + c + ".svg"), c);
-    }
+        countryEdit->addItem(QPixmap(":/resources/countries/" + c.toLower() + ".svg"), c);
```

Comment:
Could the upper values just be left over from when the db was text entry instead of a dropdown?